### PR TITLE
fix minor Japanese translations

### PR DESCRIPTION
**Pull request by zakkie**

### CODE REVIEW

File: linphone-app/assets/languages/ja.ts [hunk 1]
```diff
@@ -1028,7 +1028,7 @@
     </message>
     <message>
         <source>quit</source>
-        <translation>やめろ</translation>
```
Comment (zakkie) [inline linphone-app/assets/languages/ja.ts:1031]: "やめろ" means "Stop"

File: linphone-app/assets/languages/ja.ts [hunk 2]
```diff
@@ -1054,7 +1054,7 @@
     </message>
     <message>
         <source>menuPlayMe</source>
-        <translation>俺をプレイしろ！</translation>
```
Comment (zakkie) [inline linphone-app/assets/languages/ja.ts:1057]: F Word

File: linphone-app/assets/languages/ja.ts [hunk 3]
```diff
@@ -1363,7 +1363,7 @@
     </message>
     <message>
         <source>encryptionMandatoryLabel</source>
-        <translation>凍結は必須です</translation>
```
Comment (zakkie) [inline linphone-app/assets/languages/ja.ts:1366]: "凍結" means "Freeze"

File: linphone-app/assets/languages/ja.ts [hunk 4]
```diff
@@ -1478,7 +1478,7 @@
     </message>
     <message>
         <source>generalTitle</source>
-        <translation>将軍</translation>
```
Comment (zakkie) [inline linphone-app/assets/languages/ja.ts:1481]: "将軍" means "General of the army"

File: linphone-app/assets/languages/ja.ts (hunk 2 above)
Comment (julonexus) [reply linphone-app/assets/languages/ja.ts:1057]: Are you sure? 
https://translate.google.com/?sl=auto&tl=en&text=%E4%BF%BA%E3%82%92%E3%83%97%E3%83%AC%E3%82%A4%E3%81%97%E3%82%8D&op=translate

File: linphone-app/assets/languages/ja.ts (hunk 4 above)
Comment (julonexus) [reply linphone-app/assets/languages/ja.ts:1481]: oups :p Thanks for pointing out

File: linphone-app/assets/languages/ja.ts (hunk 2 above)
Comment (zakkie) [reply linphone-app/assets/languages/ja.ts:1057]: Japanese has a wide variety of first-person pronouns, "俺" is the male first-person pronouns.

"プレイ" has a sexual connotation, as well as meaning to play audio.

So I think most Japanese would understand it as "fxxx me" in a sexual sense.

**Comment by julonexus** [comment]
Hi, I ported your suggestion for the next release. 
We are no longer use transifex for community translation and moved to Weblate.
If you are willing to help us in the translation, we will be happy to get your contribution here : https://weblate.linphone.org/projects/linphone-desktop/secure-chat-room-4-3/ rather than github.
Regards

**Pull request closed without merging**